packages/form-js-carbon-styles/src/carbon-styles.js

```diff
@@ -271,7 +371,7 @@ const DISABLED_STYLES = css`
     }
 
     .fjs-taglist.fjs-disabled .fjs-taglist-tag {
-      background-color: var(--cds-field-02);
+      background-color: var(--cds-field-01);
```

Comment:
I think the proper way of doing this is to just use `--cds-field`, no? This way it will automatically respect the layers. This was a probably a mistake of mine when implementing it

Reply:
I added this according to the [design sheet](https://www.figma.com/file/CiTSN1vNETjIAy6W9jUIFp/Form-Editor?node-id=1034-148176&t=T2sW2Ylfu9QonnHr-0). It was supposed to be a bit less contrast in comparison to the current state. 

![image](https://github.com/bpmn-io/form-js/assets/9433996/a685e456-ede4-4d3b-b306-00b5668adbe5)

`--cds-field` would lead to the same background as for the taglist input.

Reply:
Oh actually it should be the current state, not sure why I changed it 😅 I will revert the change at all.